cypress.config.ts

```diff
@@ -58,8 +60,8 @@ export default defineConfig({
           ]),
 
         stubUserEmail: username => auth.stubEmail(username),
-        stubUser: (username, caseload) => auth.stubUser(username, caseload),
-        stubCourts: courts => whereabouts.stubCourts(courts),
+        stubUser: username => auth.stubUser(username),
+        stubCourts: () => whereabouts.stubCourts(),
         stubCourtEmailAddress: court => whereabouts.stubCourtEmailAddress(court),
```

Comment:
These functions had the wrong number of arguments. No caseload parameter on auth.stubUser() and no courts parameter on whereabouts.stubCourts() - these are statically returned from a JSON file.